templates/article/index.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}
+	{{ parent()}}
+	- Services
+{% endblock %}
+
+{% block stylesheets %}
+	{{ parent() }}
+	{{ encore_entry_link_tags('article') }}
+{% endblock %}
+
+{% block body %}
+	<main class="services-list">
+
+		<div class="header d-flex justify-content-center align-items-center my-5">
+			<h1 class="py-4 mt-0 mb-4 position-relative"><img src="{{ asset('build/images/title_underline_blue.png') }}" class="title-underline" alt="">Services</h1>
+		</div>
+			<div class="container my-5 d-flex flex-column gap-3">
+				{% for article in articles %}
+					{% if loop.index is divisible by (2) %}
```

Comment:
"is even" existe

Reply:
mais au dela de ça, il ne faudrait pas duplique complètement les cards entre pair/impair.
ça peut se gérer sans doute ne mettant qq condition inline sur certaines classes ou direct en CSS avec des nth-child(odd/even)

Reply:
mais là c'est pas assez DRY